hbase-hbtop/src/main/resources/log4j-hbtop.properties

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+log4j.rootLogger=WARN,console
```

Comment:
I wanted to avoid conflicting between a default log4j configuration file (`log4j.properties`) and this file (`log4j-hbtop.properties`). If this file is named `log4j.properties` and another `log4j.properties` exists in the classpath, we are not sure which file is read (maybe it depends on the order of loading the files). So I thought we needed a unique name for this file.

Reply:
Thank you for your advice. I will try to do that.

Reply:
I made this change in the latest commit. Can you please re-review it?